ansible/roles/proxmox_cluster/README.md

```
@@ -0,0 +1,155 @@
+pvecm status | grep -i quorum
+```
+
+## Testing
+
+```bash
+# Syntax check
+ansible-playbook --syntax-check playbooks/initialize-cluster.yml
+
+# Check mode (dry run) - safe for testing
+ansible-playbook playbooks/initialize-cluster.yml --check --diff
+
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)